src/Service.Tests/DatabaseSchema-MsSql.sql

```diff
@@ -66,7 +66,7 @@ CREATE TABLE books(
 
 CREATE TABLE players(
     id int IDENTITY(5001, 1) PRIMARY KEY,
-    name varchar(max) NOT NULL,
+    [name] varchar(max) NOT NULL,
```

Comment:
brackets because name is a reserved keyword?

Reply:
yes.